app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,32 @@ def show
     @likes = Post.find(params[:id]).likes
     @comments = Post.find(params[:id]).comments
   end
+
+  # GET /posts/new
+  def new
+    @post = Post.new
+  end
+
+  # POST /posts or /posts.json
+  def create
+    @post = Post.new(post_params.merge(comments_counter: 0, likes_counter: 0, author_id: current_user.id))
```

Comment:
- [ ] There is an error when creating a post indicating the id is nil
![image](https://user-images.githubusercontent.com/45460192/200061917-531c29c2-1ee9-4531-b8f6-d517321fdd25.png)

Reply:
Hi @Lucas-Masaba, the create post form works fine on my side. please check the below screenshots 
![image](https://user-images.githubusercontent.com/22744775/200106138-f973ea8b-deac-45cd-b81a-a8b8ff6c54b7.png)
after submit 
![image](https://user-images.githubusercontent.com/22744775/200106179-9b1f1af2-f4fb-4a4e-a758-a38580fdb7da.png)